Resources/views/CRUD/action_buttons.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% spaceless %}
-    {% for item in admin.getActionButtons(action, object|default(null)) %}
+    {% for item in admin.getActionButtons(action, (object is defined and object is not null)?object:null ) %}
```

Comment:
does this really change anything?

If the object is defined we already take it, if not we set null by default, IMO your code does exactly the same, correct me if I am wrong